docs/interpreter/python.md

```diff
@@ -232,6 +232,70 @@ SELECT * FROM rates WHERE age < 40
 Otherwise it can be referred to as `%python.sql`
 
 
+## IPython Support
+
+IPython is more powerful than the default python interpreter with extra functionality. You can use IPython2 or IPython3 which depends on which python you set `zeppelin.python`.
```

Comment:
`You can use IPython2 or IPython3 which depends on which python you set zeppelin.python.`
should this say `You can use IPython with Python2 or Python3 which depends on which python you set zeppelin.python.`

IPython has separate versions independent of python2 or python3

Reply:
Fixed